src/d3a/d3a_core/simulation.py

```diff
@@ -351,20 +356,15 @@ def _execute_simulation(self, slot_resume, tick_resume, console=None):
             mbs_used = process.memory_info().rss / 1000000.0
             log.debug(f"Used {mbs_used} MBs.")
 
-            simulation_time_counter = time.time()
+            tick_time_counter = time()
+
             for tick_no in range(tick_resume, config.ticks_per_slot):
-                tick_start = time.time()
-                self._handle_paused(console, tick_start)
+                self._handle_paused(console, tick_time_counter)
 
                 # reset tick_resume after possible resume
                 tick_resume = 0
-                log.trace(
-                    "Tick %d of %d in slot %d (%2.0f%%)",
-                    tick_no + 1,
-                    config.ticks_per_slot,
-                    slot_no + 1,
-                    (tick_no + 1) / config.ticks_per_slot * 100,
-                )
+                log.trace(f"Tick {tick_no + 1} of {config.ticks_per_slot} in slot "
+                          f"{slot_no + 1} ({(tick_no + 1) / config.ticks_per_slot * 100})")
```

Comment:
Trivial since it is a trace log, but in order to mimic the same behavior you need to limit the floating point precision.